src/shred_network/repair_service.zig

```diff
@@ -495,6 +545,10 @@ pub const RepairPeerProvider = struct {
     }
 };
 
+/// The maximum time to wait between repair loop iterations.
+/// Sort of like a timeout for repair responses.
+const MAX_REPAIR_DELAY_MILLIS = 1_000;
```

Comment:
Duration? -- feel like we should push for always using sig durations in constants, wdyt? -- maybe add to contributing too?

Reply:
Harold had a similar comment on some other code. It was a bit complicated since our time types did not have sufficient support for what I needed. Sometimes it's more straightforward to work with integers than opaque time structs. But I added the necessary methods for what he requested. I'll see about these other ones too.

Reply:
I know you resolved this but I made some changes already. Also tried to document the code better since it's a bit arbitrary. c323f03d45f332b533ef2a96d7ba81144f6325b4